### ISSUE TITLE
1.17 - 1.18 not working in VS 2017 15.5

### ISSUE THREAD

**Issue by lmdg** [frhagn/Typewriter#253]
Hi there,
Was really excited when I found this. But I've spent a couple of hours trying to get this to work on VS Community 2017 15.5 (also tried VS 2017 15.0 and the 1.17 and 1.17.1 Typewriter versions) but nothing seems to be running. After installing the extension the file type icon on the solution explorer for the .tst changes to 'TS' but nothing else. There's also no TST type under 'Add new item'->Web. Are there any VS package installation requirements? Thanks again.

**Comment by frhagn** [comment]
Hi @lmdg
That's wierd, everything seems to be working fine for me... 
What project type are you using (Typewriter doesn't work with Web _Site_ Projects)? Is there any output from Typewriter in the Output window?

**Comment by lmdg** [comment]
Thanks so much for the quick reply @frhagn ,
I guess it might be a web site? This is a quick excerpt of my csproj:

`<Project Sdk="Microsoft.NET.Sdk.Web">

  <PropertyGroup>
    <TargetFramework>netcoreapp2.0</TargetFramework>
  </PropertyGroup>
...
`
What type of project would I have to make it so it's compatible?
And apologies, I couldn't see any output from Typewriter, tried to find it (it's my first time working with VS) before I went on a VS-install/uninstall-different-versions-spree.
Thanks again for all of the help.

**Comment by lmdg** [comment]
It seems this was because I was using a project that started from an empty ASP.NET Core Web App. When I went to create a new Web App (after I had installed Typewriter) and chose Web Api template TypeWriter then kicked in (including on the original project). I don't know if it was the new project or using the Web api template that triggered it but it's working great now. Thanks again for the help @frhagn .
